src/components/input-folder-checkbox.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ChangeEvent } from 'react';
+import { useTranslator } from '../hooks';
+
+import {
+  Checkbox,
+  FormControlLabel,
+  FormGroup,
+  FormHelperText
+} from '@mui/material';
+
+export function PackageInputFolderControl(props: {
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  inputFile: string;
+}): JSX.Element {
+  const trans = useTranslator('jupyterlab');
+  const inputFilePath = props.inputFile.split('/');
+  inputFilePath.pop();
+
+  let helperText: string;
+  if (inputFilePath.length) {
+    const inputFolder = `/${inputFilePath.join('/')}`;
+    helperText = trans.__(
+      'The scheduled job will have access to all files under %1',
+      inputFolder
+    );
+  } else {
+    helperText = trans.__(
+      "The scheduled job will have access to all files under input file's folder"
```

Comment:
```suggestion
      "The scheduled job will have access to all files under the input file's folder"
```

Please also update the screen shot in the docs.